dockstore-client/src/main/java/io/dockstore/client/cli/nested/ApiClientExtended.java

```diff
@@ -0,0 +1,78 @@
+package io.dockstore.client.cli.nested;
+
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Form;
+import javax.ws.rs.core.MediaType;
+
+import io.swagger.wes.client.ApiClient;
+import io.swagger.wes.client.ApiException;
+import org.glassfish.jersey.media.multipart.FormDataBodyPart;
+import org.glassfish.jersey.media.multipart.FormDataContentDisposition;
+import org.glassfish.jersey.media.multipart.MultiPart;
+
+public class ApiClientExtended extends ApiClient {
+
+
+    public void createBodyPart(MultiPart multiPart, String key, Object formObject) {
+        if (formObject instanceof File) {
+            File file = (File)formObject;
+            FormDataContentDisposition contentDisp = FormDataContentDisposition.name(key)
+                    .fileName(file.getName()).size(file.length()).build();
+            multiPart.bodyPart(new FormDataBodyPart(contentDisp, file, MediaType.APPLICATION_OCTET_STREAM_TYPE));
+        } else {
+            FormDataContentDisposition contentDisp = FormDataContentDisposition.name(key).build();
+            multiPart.bodyPart(new FormDataBodyPart(contentDisp, parameterToString(formObject)));
+        }
+    }
+
+    /**
+     * Serialize the given Java object into string entity according the given
+     * Content-Type (only JSON is supported for now).
+     * @param obj Object
+     * @param formParams Form parameters
+     * @param contentType Context type
+     * @return Entity
+     * @throws ApiException API exception
+     */
+    @Override
+    public Entity<?> serialize(Object obj, Map<String, Object> formParams, String contentType) throws ApiException {
+        Entity<?> entity;
+        if (contentType.startsWith("multipart/form-data")) {
+            MultiPart multiPart = new MultiPart();
+            for (Map.Entry<String, Object> param: formParams.entrySet()) {
+                if (param.getValue() instanceof File) {
+                    createBodyPart(multiPart, param.getKey(), param.getValue());
+                    //File file = (File) param.getValue();
```

Comment:
Add comment, think its the same deal as earlier. 
Maybe add in a github ticket number DOCK-X or the URL https://github.com/ga4gh/dockstore/issues/2123 so that it would be trivial for the following developer to search for and turn on everything

Reply:
Removed unneeded commented code